unity_common/src/Commands/UnityDrushCommands.php

```diff
@@ -26,38 +26,31 @@ public function __construct() {
   }
 
   /**
-   * Drush command import blocks, taxonomies and menus  using structure_sync.
+   * Drush command import blocks and taxonomies using structure_sync.
    *
-   * @param string $option
-   *   Argument to select 'safe', 'full' or 'force'.
    *
-   * @command import-all-if-installed
+   * @command import-blocks-taxonomies
    */
-  public function importAllIfInstalled($option = 'safe') {
+  public function importBlocksTaxonomies() {
     // Only import if the structure_sync module is installed.
     if (\Drupal::moduleHandler()->moduleExists('structure_sync')) {
       $config = \Drupal::config('structure_sync.data');
-      // Import blocks if necessary.
+      // Import blocks if necessary, always use 'full' so that any block
+      // changes in config will be reflected on the site.
```

Comment:
Does Darren not sometimes make changes to the blocks on the live sites? Would this not override them?

Reply:
Yes it would, if he does that then he should tell us and we should export the changes shouldn't we ?
(Just the same as with any change on a live site)

Reply:
What I was thinking here is that the only proper way to add a new block to a live site is to add it in config, we shouldn't ever be editing live sites directly should we ?
I suppose that I could change it to 'safe' for blocks which would mean that we could add new blocks through config, but couldn't make any changes to existing blocks through config. But I don't really like that idea, as again, you shouldn't really have to log on to a live site to make a change to a block.

Reply:
I have tried not to add block content/menu links to structure sync because Darren likes to make changes on live. So the block is created in config but the content remains in the database?

Reply:
The block should be in the config, but the content is in the database so that it can be edited on live, new blocks will still require being added through the config